UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Controllers/BaseController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Mvc;
+
+namespace UKHO.D365CallbackDistributorStub.API.Controllers
+{
+    [ExcludeFromCodeCoverage]
+    [ApiController]
+    [Route("api/[controller]")]
+    public abstract class BaseController<T> : ControllerBase
+    {
+        protected IActionResult GetOkResponse()
+        {
+            return Ok();
+        }
+
+        protected IActionResult GetBadRequestResponse()
```

Comment:
Change name from GetBadRequestResponse() to BadRequestResponse()

Reply:
Renamed